ts/features/walletV3/onboarding/components/WalletOnboardingError.tsx

```diff
@@ -1,42 +1,79 @@
 /* eslint-disable functional/immutable-data */
 import * as React from "react";
-import * as O from "fp-ts/lib/Option";
-import { pipe } from "fp-ts/lib/function";
 
-import { OnboardingOutcome } from "../types";
+import { OnboardingOutcome, OnboardingOutcomeEnum } from "../types";
 import I18n from "../../../../i18n";
-import { GenericErrorComponent } from "../../common/components/GenericErrorComponent";
+import { OperationResultScreenContent } from "../../../../components/screens/OperationResultScreenContent";
+import { openWebUrl } from "../../../../utils/url";
+import {
+  ONBOARDING_FAQ_ENABLE_3DS,
+  ONBOARDING_OUTCOME_ERROR_PICTOGRAM
+} from "../utils";
 
 type WalletOnboardingErrorProps = {
   onClose: () => void;
-  outcome?: OnboardingOutcome;
+  outcome: Exclude<OnboardingOutcome, `${OnboardingOutcomeEnum.SUCCESS}`>;
 };
 
-const getOutcomeMessage = (outcome?: OnboardingOutcome): string =>
-  pipe(
-    outcome,
-    O.fromNullable,
-    O.fold(
-      () => I18n.t("wallet.onboarding.generalError"),
-      outcome => `Outcome ricevuto: ${outcome}` // TODO: Replace with the desired message to be defined
-    )
-  );
+type OnboardingErrorEnumKey = Exclude<
```

Comment:
isn't this type a duplication of the `outcome` spec at L15?

Reply:
No, this is not a type duplication. The `outcome` value at line 15 is an `OnboardingOutcome` type which is a string indicating the value of the `OnboardingOutcomeEnum`, meanwhile the `OnboardingErrorEnumKey` is the key of the enum and not his value.

To do an example, this enum definition:

```typescript
enum OnboardingOutcomeEnum {
    SUCCESS = "0",
    GENERIC_ERROR = "1",
    ...
}
```

The `OnboardingErrorEnumKey` are the enum keys (in this case, SUCCESS, GENERIC_ERROR and so on), the `OnboardingOutcome` is the enum value (in this case, "0","1", and so on)